.github/workflows/tsan-suppressions_3.13.txt

```diff
@@ -40,7 +40,3 @@ race:gemm_oncopy
 # https://github.com/python/cpython/issues/132245
 race:split_keys_entry_added
 race_top:dict_dict_merge
-
-# https://github.com/python/cpython/issues/132013
```

Comment:
It was backported: https://github.com/python/cpython/issues/132013#issuecomment-2798952335